### PR TITLE
Remove constraint for normal priorities.

### PR DESCRIPTION
**Pull request by arkhi**
This is following retrospective meeting for sprint 16.

@juwai/all: Unleash your thumbs!

### CODE REVIEW

**Comment by mark-juwai** [comment]
👍

**Comment by piadelosreyes** [comment]
👍

File: workflow-issue-priority.md [hunk 1]
```diff
@@ -35,8 +35,6 @@ Finish the task presently assigned to you, commit, then work on the issue.
 
 Finish your story, commit, but don’t start any new story before this issue is fixed.
```
Comment (tjoelsson) [inline workflow-issue-priority.md]: Does this still apply?

Comment (arkhi) [inline workflow-issue-priority.md]: @mryder?

Comment (mryder) [inline workflow-issue-priority.md]: I'm okay with this as written (and for the future) but I understand if we need to do something to "ease" into this during this sprint or next. I'm sure that we're a few too many High tickets away from having this be immediately feasible.

I'm also okay with changing the work "fixed" into "addressed" and running with that for a while. If that turns out not to work we can change it back to "fixed" at a later date. Thoughts?

Comment (arkhi) [inline workflow-issue-priority.md]: Thanks. I updated the sentence.

**Comment by tjoelsson** [comment]
👍

**Comment by arkhi** [comment]
@mark-juwai & @piadelosreyes: I don’t want to take more of your time, so I’ll merge and will address comments if you leave more.

Thanks!

**Comment by arkhi** [comment]
Anybody who can merge is welcome to do so. Thanks!

**Comment by mark-juwai** [comment]
👍  I'm not allowed to do that :)

**Comment by mryder** [comment]
@mark-juwai that should be fixed now.

**Comment by mark-juwai** [comment]
Thanks, @mryder

**Pull request merged**